servers/rendering_server.h

```diff
@@ -1838,8 +1842,10 @@ class RenderingServer : public Object {
 
 	virtual Size2i get_maximum_viewport_size() const = 0;
 
+#ifdef RD_ENABLED
 	RenderingDevice *get_rendering_device() const;
 	RenderingDevice *create_local_rendering_device() const;
+#endif // RD_ENABLED
```

Comment:
These also break encapsulation, but I guess solving that will be trickier / not possible.